pkg/event/event.go~

```diff
@@ -0,0 +1,134 @@
+// Copyright 2013 The Prometheus Authors
```

Comment:
is seems like a few `*~` files have snuck in, would you mind removing them and adding a gitignore pattern (or possibly better, add this to your own global gitignore, since it's your environment that creates them?)